dbms/include/DB/Columns/ColumnVector.h

```diff
@@ -265,6 +265,16 @@ class ColumnVector final : public IColumn
 		res = typename NearestFieldType<T>::Type(data[n]);
 	}
 
+	const T & get(size_t n) const
```

Comment:
it's very strange that there are methods get64, get(n, Field), getDataAt(n) -> StringRef but there is no method which return's T.